.github/workflows/main.yml

```
@@ -3,10 +3,23 @@ on:
   push:
     branches:
       - main
+      - typos
```

Not sure about the indentation, but this could potentially trigger the action?
```suggestion
  pull_request:
    branches:
      - main
```